### PR TITLE
Create problem_006.java

### PR DESCRIPTION
**Pull request by Awani-Shukla**
**Fixes issue:**

**Changes:**

### CODE REVIEW

File: code/online_challenges/src/project_euler/problem_006/problem_006.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class Euler {
+	public static void main(String[] args) {
+        int sum = 0;
+        int sqsum = 0;
+        for (int i=1; i<=100; i++) {
+            sqsum += i*i;
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_006/problem_006.java]: Add spaces around all operators.

File: code/online_challenges/src/project_euler/problem_006/problem_006.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class Euler {
+	public static void main(String[] args) {
+        int sum = 0;
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_006/problem_006.java]: Use consistent indentation.

File: code/online_challenges/src/project_euler/problem_006/problem_006.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class Euler {
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_006/problem_006.java]: Call the class something like Problem006.

**Comment by arnavb** [comment]
(Sorry for repeat review, internet issues)

File: code/online_challenges/src/project_euler/problem_006/problem_006.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class Problem006{
+	public static void main(String[] args) {
+		int sum = 0;
+		int sqsum = 0;
+		for (int i = 1; i <= 100; i ++) {
+			sqsum += i * i;
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_006/problem_006.java]: Fix indentation in this loop.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**